src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -482,9 +482,14 @@ GetDistributeObjectOps(Node *node)
 					return &Type_AlterObjectSchema;
 				}
 
+				case OBJECT_TABLE:
```

Comment:
small change required here.

I think the case's are deliberately kept in alphabetic order. Can we move `OBJECT_TABLE` above `OBJECT_TYPE`.